File: spring-user/src/main/java/com/example/springuser/dto/ResultCode.java
package com.example.springuser.dto;

public enum ResultCode {
    SUCCESS("정상 처리 되었습니다."),
    ERROR("에러가 발생했습니다."),
    FORBIDDEN("권한이 없습니다."),
    NOT_FOUND("리소스를 찾을 수 없습니다.");

    private final String msg;

    // 생성자
    ResultCode(String msg) {
        this.msg = msg;
    }

    // msg 값을 반환하는 getter 메서드
    public String getMsg() {
        return msg;
    }
}

File: spring-kotlin-user/src/main/kotlin/com/example/kotlin/user/common/status/ResultCode.kt
package com.example.kotlin.user.common.status

enum class ResultCode(val msg: String) {
    SUCCESS("정상 처리 되었습니다."),
    ERROR("에러가 발생했습니다."),
    FORBIDDEN("권한이 없습니다."),
    NOT_FOUND("리소스를 찾을 수 없습니다."),
}